client/src/app/site/pages/meetings/pages/participants/pages/participant-import/definitions/index.ts

```diff
@@ -1,23 +1,33 @@
-import { marker as _ } from '@colsen1991/ngx-translate-extract-marker';
+import { userHeadersAndVerboseNames } from 'src/app/domain/models/users/user.constants';
 import { GeneralUser } from 'src/app/gateways/repositories/users';
 
 export const participantHeadersAndVerboseNames: { [key in keyof GeneralUser]?: any } = {
-    title: _(`Title`),
-    first_name: _(`Given name`),
-    last_name: _(`Surname`),
-    email: _(`Email`),
-    member_number: _(`Member number`),
+    ...userHeadersAndVerboseNames,
     structure_level: `Structure levels`,
     groups: `Groups`,
     number: `Participant number`,
     vote_weight: `Vote weight`,
-    gender: _(`Gender`),
-    pronoun: _(`Pronoun`),
-    username: _(`Username`),
-    default_password: _(`Initial password`),
-    is_active: _(`Active`),
-    is_physical_person: _(`Natural person`),
     is_present: `Is present`,
-    saml_id: _(`SSO identification`),
     comment: `Comment`
 };
+
+export const participantColumnsWeight: { [key in keyof GeneralUser]?: any } = {
+    title: 1,
+    first_name: 2,
+    last_name: 3,
+    email: 4,
+    member_number: 5,
+    structure_level: 6,
+    groups: 7,
+    number: 8,
+    vote_weight: 9,
+    gender: 10,
+    pronoun: 11,
+    username: 12,
+    default_password: 13,
+    is_active: 14,
+    is_physical_person: 15,
+    is_present: 16,
+    saml_id: 17,
+    comment: 18
+};
```

Comment:
Using an array with the index being the weight may also work, would spare you the work of having to re-sort the keys every time you need the keys in-order

Reply:
Now use an array.